agents/backport_agent.py

```diff
@@ -288,6 +289,12 @@ def get_prompt() -> str:
       Everything from the previous attempt has been reset. Start over, follow the instructions from the start
       and don't forget to fix the issue.
       {{/build_error}}
+      {{#reproducer_failed}}
+      The reproducer failed after the first backport attempt.
```

Comment:
Like I said elsewhere, this is confusing. We want the reproducer to fail - if it fails, the fix worked. But only if it didn't fail before applying the fix, if it did, it's a bogus reproducer. I would use the term `reproducer test case`, and run it twice, first without the new build, expected to fail, and then with the new build, expected to succeed.